backend/canisters/local_group_index/impl/src/updates/c2c_delete_group.rs

```diff
@@ -21,6 +21,7 @@ fn c2c_delete_group_impl(args: Args, state: &mut RuntimeState) -> Response {
     }
 }
 
+// TODO make this retry upon failure
 async fn delete_canister(canister_id: CanisterId) {
```

Comment:
Can't you just use the fire_and_forget_handler?

Reply:
It's not quite that simple because it needs to call stop and then delete.
But we should be able to do something like that yeah.